examples/orderbook/create-bid-with-nextjs/playwright.config.ts

```diff
@@ -0,0 +1,30 @@
+import { defineConfig, devices } from "@playwright/test";
+
+export default defineConfig({
+    testDir: "./tests",
+    fullyParallel: true,
+    forbidOnly: !!process.env.CI,
+    retries: process.env.CI ? 2 : 0,
+    workers: process.env.CI ? 1 : undefined,
```

Comment:
The playwright config base used for examples was recently updated to work a bit better on CI. The changes made were:
`workers: "80%"`,
and
`webServer.command: "yarn start"`

Could those changes be applied to this PR?